test_sets/External_Consent_Status_ON.txt

```diff
@@ -0,0 +1,33 @@
+./test_scripts/Polices/External_UCS/External_Consent_ON/001_ATF_Policies_External_Consent_ON_disallowed_by_external_consent_entities_on_data_consent_disallowed.lua Check_that_SDL_returns_DISALLOWED_when_disallowed_by_external_consent_entities_off_exists_externalConsentStatus_ON_and_data_consent_disallowed
+./test_scripts/Polices/External_UCS/External_Consent_ON/002_ATF_Policies_External_Consent_ON_disallowed_by_external_consent_entities_on_user_consent_allowed.lua Check_that_SDL_returns_SUCCESS_when_disallowed_by_external_consent_entities_off_exists_externalConsentStatus_ON_and_functional_group_allowed_data_consent_allowed
+./test_scripts/Polices/External_UCS/External_Consent_ON/003_ATF_Policies_External_Consent_ON_disallowed_by_external_consent_entities_on_user_consent_disallowed.lua Check_that_SDL_returns_USER_DISALLOWED_when_disallowed_by_external_consent_entities_off_exists_externalConsentStatus_ON_and_functional_group_disallowed_by_user
+./test_scripts/Polices/External_UCS/External_Consent_ON/004_ATF_Policies_External_Consent_ON_disallowed_by_external_consent_entities_off_data_consent_disallowed.lua Check_that_SDL_returns_DISALLOWED_when_disallowed_by_external_consent_entities_on_exists_externalConsentStatus_ON_and_data_consent_disallowed
+./test_scripts/Polices/External_UCS/External_Consent_ON/005_ATF_Policies_External_Consent_ON_disallowed_by_external_consent_entities_off_user_consent_allowed.lua Check_that_SDL_returns_USER_DISALLOWED_when_disallowed_by_external_consent_entities_on_exists_externalConsentStatus_ON_and_functional_group_allowed_data_consent_allowed
+./test_scripts/Polices/External_UCS/External_Consent_ON/006_ATF_Policies_External_Consent_ON_disallowed_by_external_consent_entities_off_user_consent_disallowed.lua Check_that_SDL_returns_USER_DISALLOWED_when_disallowed_by_external_consent_entities_on_exists_externalConsentStatus_ON_and_functional_group_disallowed_by_user
```

Comment:
@dboltovskyi, name of script is for disallowed_by_external_consent_entities_off, but in description is specified disallowed_by_external_consent_entities_on. 
Please clarify